tests/py/test_ptq_to_backend.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import trtorch
+from trtorch.logging import *
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+import torchvision
+import torchvision.transforms as transforms
+from model_test_case import ModelTestCase
+
+
+class TestAccuracy(ModelTestCase):
+
+    def setUp(self):
+        self.input = torch.randn((1, 3, 32, 32)).to("cuda")
+        self.testing_dataset = torchvision.datasets.CIFAR10(root='./data',
+                                                            train=False,
+                                                            download=True,
+                                                            transform=transforms.Compose([
+                                                                transforms.ToTensor(),
+                                                                transforms.Normalize((0.4914, 0.4822, 0.4465),
+                                                                                     (0.2023, 0.1994, 0.2010))
+                                                            ]))
+
+        self.testing_dataloader = torch.utils.data.DataLoader(self.testing_dataset,
+                                                              batch_size=1,
+                                                              shuffle=False,
+                                                              num_workers=1)
+        self.calibrator = trtorch.ptq.DataLoaderCalibrator(self.testing_dataloader,
+                                                           cache_file='./calibration.cache',
+                                                           use_cache=False,
+                                                           algo_type=trtorch.ptq.CalibrationAlgo.ENTROPY_CALIBRATION_2,
+                                                           device=torch.device('cuda:0'))
+
+        self.spec = {
+            "forward":
+                trtorch.TensorRTCompileSpec({
+                    "input_shapes": [[1, 3, 32, 32]],
+                    "op_precision": torch.int8,
+                    "calibrator": self.calibrator,
+                    "device": {
+                        "device_type": trtorch.DeviceType.GPU,
+                        "gpu_id": 0,
+                        "dla_core": 0,
+                        "allow_gpu_fallback": False,
+                    }
+                })
+        }
+
+    def compute_accuracy(self, testing_dataloader, model):
+        total = 0
+        correct = 0
+        loss = 0.0
+        class_probs = []
+        class_preds = []
+
+        with torch.no_grad():
+            idx = 0
+            for data, labels in testing_dataloader:
+                data, labels = data.cuda(), labels.cuda(non_blocking=True)
```

Comment:
data.to(device) maybe to avoid warnings ?

Reply:
this shouldnt throw warnings since its using the new api. the .cuda api wasnt deprecated it was just the async flag